lib/nodeschnaps/node/binding/fs.js

```diff
@@ -340,9 +342,15 @@ fs.fstat = function fstat(fd, callback)
         var fileHandle = fdList.get(fd);
 
         if (JavaFiles !== null && (JavaFiles.getFileAttributeView instanceof Function)) {
+            var view = JavaBasicFileAttributeView;
+
+            if (jsEngine.getEngineId() !== jsEngine.ids.ENGINE_RHINO) {
```

Comment:
This will not work. Rhino fails in the this case with:
```
line 345: Java class "java.nio.file.attribute.BasicFileAttributeView" has no public
 instance field or method named "class".
```